extension/js/inspector/app/modules/Activity/models/ActivityNode.js

```diff
@@ -11,7 +11,16 @@ define([
   */
   var _filterTreeNode = function (treeNode) {
     return treeNode.event && treeNode.event.get('listeners').length || treeNode.nodes.length;
-  }
+  };
+
+  var _filterForAction = function (treeNode, actionId) {
+    return (treeNode.event && treeNode.event.get('listeners').length || treeNode.nodes.length) && (treeNode.event && treeNode.event.get('actionId') === actionId) || isAction(treeNode) && treeNode.nid === actionId;
+  };
+
+  var isAction = function(node) {
```

Comment:
oh man, we should probably create an `isAction` field on the collection models so that this is easier.